cdc/owner/changefeed.go

```diff
@@ -339,11 +339,18 @@ func (c *changefeed) tick(ctx cdcContext.Context, captures map[model.CaptureID]*
 		return nil
 	}
 
+	// if redo is not enabled, use the barrierTs as the `newResolvedTs`
+	// not need to wait the slowest table
+	if !c.redoManager.Enabled() {
+		newResolvedTs = barrierTs
+	}
+
```

Comment:
global resolved ts and barrier ts have different meaning, so I suggest use two variables instead of  the one in the following codes